board/main.c

```diff
@@ -778,23 +783,22 @@ void TIM1_BRK_TIM9_IRQ_Handler(void) {
     loop_counter++;
     loop_counter %= 8U;
   }
-  TIM9->SR = 0;
+  HEARTBEAT_TIMER->SR = 0;
 }
 
-#define MAX_FADE 8192U
+
 int main(void) {
+  clock_init();
```

Comment:
again, why the move? this needs to be tested well

Reply:
Already tested, but will do more and will get more info. But always had a rule to init clock first and peripherals - second. (and interrupts are off from the start, but understand that precaution)

Reply:
are you sure interrupts are off from the start? something tells me that they might be enabled if you reset out of DFU